### PR TITLE
Add !formatting command

### PR DESCRIPTION
**Pull request by nickmccurdy**
image

### CODE REVIEW

**Comment by nickmccurdy** [comment]
That's what I was trying to suggest with "create churn from style debates" vs."follow best practices. I wouldn't want to get too wordy with in-depth explanations, but maybe we could link to a docs page regarding this? https://prettier.io/docs/en/index.html

**Comment by vcarl** [comment]
In the interest of getting this out I'm gonna merge. Addressing anti-formatter arguments is a nice-to-have to be sure.
